runtime/dancebox/Cargo.toml

```diff
@@ -234,7 +234,6 @@ try-runtime = [
 	"pallet-invulnerables/try-runtime",
 	"pallet-maintenance-mode/try-runtime",
 	"pallet-migrations/try-runtime",
-	"pallet-migrations/try-runtime",
```

Comment:
reason for this?

Reply:
It's duplicated, right? The line above is the same

Reply:
ah right